src/utils/pagination/formatPaginatedDataFromServer.spec.js

```diff
@@ -3,12 +3,10 @@ import * as objectUtils from '../objects';
 import formatPaginatedDataFromServer from './formatPaginatedDataFromServer';
 
 describe('utils/pagination/formatPaginatedDataFromServer', function() {
-  beforeEach(function() {
-    this.sandbox = sinon.createSandbox();
-  });
+  beforeEach(function() {});
```

Comment:
There's a few empty blocks like this -- can you remove them?

Reply:
lol that is pretty damn good.